system-administrators/deployment/network-deployment.md

```diff
@@ -28,7 +37,11 @@ Next the _compose and configuration profiles_ must be generated using a script,
 
 ### Step 1
 
-For step 1, the network name `${MEDCO_SETUP_NETWORK_NAME}` should be common to all the nodes. `${MEDCO_SETUP_NODE_DNS_NAME}` corresponds to the machine domain name where the node is being deployed. As mentioned before the different parties should have agreed beforehand on the members of the network, and assigned an index `${MEDCO_SETUP_NODE_IDX}` to each different node to construct its UID \(starting from `0`, to `n-1`, `n` being the total number of nodes\).
+For step 1, the network name `${MEDCO_SETUP_NETWORK_NAME}` should be common to all the nodes. `${MEDCO_SETUP_NODE_DNS_NAME}` corresponds to the machine domain name where the node is being deployed. As mentioned before the different parties should have agreed beforehand on the members of the network, and assigned an index `${MEDCO_SETUP_NODE_IDX}` to each different node to construct its UID \(starting from `0`, to `n-1`, `n` being the total number of nodes\). The nodes id's need to contain 3 digits (`0` translates to `000`, `1` to `001`).
```

Comment:
The padding to 3 digits should be done automatically by the script, thus not needed to specify here. Did you have a specific issue regarding this?

Reply:
I will confirm with Jules on that. If I remember correctly this was problematic. I'll be back with an answer soon.

Reply:
@Fasjul